apps/vs-code-extension/src/commands/run-jv.ts

```diff
@@ -0,0 +1,78 @@
+// SPDX-FileCopyrightText: 2023 Friedrich-Alexander-Universitat Erlangen-Nurnberg
+//
+// SPDX-License-Identifier: AGPL-3.0-only
+
+// eslint-disable-next-line unicorn/prefer-node-protocol
+import { strict as assert } from 'assert';
+import { exec } from 'node:child_process';
+import { readFileSync } from 'node:fs';
+import path from 'node:path';
+
+import { isRunJayveeCodeLensPayload } from '@jvalue/jayvee-language-server';
+import { type ExtensionContext, window } from 'vscode';
+
+export async function runJayveeCommand(
+  payload: unknown,
+  context: ExtensionContext,
+) {
+  try {
+    await throwOnMismatchingJayveeVersion(context);
+  } catch (e: unknown) {
+    const errorMsg = e instanceof Error ? e.message : 'unknown error';
+    await window.showErrorMessage(`Error executing jv: ${errorMsg}`);
+  }
+
+  assert(
+    isRunJayveeCodeLensPayload(payload),
+    `Payload ${JSON.stringify(
+      payload,
+    )} is not a valid RunJayveeCodeLensPayload`,
+  );
+  const filePath = payload.filePath;
+
+  const shell = window.createTerminal('Run Jayvee');
+  shell.sendText(`jv ${filePath} --pipeline ${payload.pipelineName}`);
```

Comment:
Would put a `-d` in there, no?

Reply:
Mhm, depends. We could also add a second code action `Debug (once)`?
But in general, we should offer an option to debug, I agree!

Reply:
Added in [3a61993](https://github.com/jvalue/jayvee/pull/606/commits/3a61993b7ceb13680eec75d64aacca3ea2ea00b3)